src/main/java/core/basesyntax/service/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    private FruitStorageDao storageDao;
+
+    public ReturnOperation(FruitStorageDao storageDao) {
```

Comment:
It's good practice to validate the input parameters of a method. Consider checking if storageDao is not null in the constructor to avoid potential NullPointerExceptions when calling methods on storageDao.